mixin/repo.go

```diff
@@ -178,6 +178,13 @@ func runInitRepo(cmd *cobra.Command, args []string) {
 		fail(err)
 	}
 
+	// save working directory so we can come back,
+	// this function may be called other than by commandline
+	wd, err := os.Getwd()
+	if err != nil {
+		fail(err)
+	}
+
 	err = os.Chdir(mixWS)
 	if err != nil {
```

Comment:
Do we want a
                defer func { _ = os.Chdir(wd)
here, rather than an explicit chdir at the end?